code/test.py

```
@@ -0,0 +1,11 @@
+# TEST
+
+def main(x,y,z) -> str:
```

Ruff (0.8.2)

Missing type annotation for function argument `x`

(ANN001)